plugins/modules/postgresql_user.py

```diff
@@ -920,6 +950,56 @@ def add_comment(cursor, user, comment, check_mode):
         return False
 
 
+def user_configuration(cursor, module, user, configuration):
+    """Updates the user's configuration parameters if necessary."""
+    current_config_query = "SELECT rolconfig FROM pg_roles WHERE rolname = %(user)s;"
+    cursor.execute(current_config_query, {"user": user})
+    current_config = cursor.fetchone()
+    current_config_dict = {}
+    changed = False
+
+    if current_config is None:
+        module.fail_json(msg="Can't find user %(user)s in 'pg_roles'" % {"user": user})
+
+    if current_config['rolconfig'] is not None:
+        for item in current_config['rolconfig']:
+            tokens = item.split("=", 1)
+            current_config_dict[tokens[0]] = tokens[1]
+
+    c = configuration.copy()
+    reset = []
+
+    # check each item in the current configuration
+    for key, value in current_config_dict.items():
+        # we already have the correct setting
+        if key in c and value == c[key]:
+            # so we can remove it from the list
+            del c[key]
+        # if the key is not in the list of settings we want
+        elif key not in c:
+            # we will reset it on the database
+            reset.append(key)
+        # if the setting is not in the db or has the wrong value, it will get updated
+
+    try:
+        # It seems psycopg's prepared statements don't work with 'ALTER ROLE' as this point.
+        # This is vulnerable to SQL-injections (added to docs) but I don't see a better way to do this.
+        for item in reset:
+            query = 'ALTER ROLE "%(user)s" RESET "%(key)s";' % {"user": user, "key": item}
+            cursor.execute(query)
+            executed_queries.append(query)
```

Comment:
```suggestion
            executed_queries.append(query)
            cursor.execute(query)
```
i suggest appending first as it can be helpful when debugging, if you don't mind